Sources/Private/Experimental/ExperimentalAnimationLayer.swift

```diff
@@ -75,6 +80,13 @@ final class ExperimentalAnimationLayer: CALayer {
     for animationLayer in animationLayers {
       animationLayer.setupAnimations(context: context)
     }
+
+    // We also set up a placeholder animation to track the current animation progress
+    let animationProgressTracker = CABasicAnimation(keyPath: #keyPath(animationProgress))
+    animationProgressTracker.configureTiming(with: context)
+    animationProgressTracker.fromValue = 0
+    animationProgressTracker.toValue = 1
+    add(animationProgressTracker, forKey: key)
```

Comment:
Why do we need to pass this key through from the caller vs. having a static key? Might be helpful to document

Reply:
`AnimationView.isAnimationPlaying` is implemented by checking `animationLayer?.animation(forKey: activeAnimationName) != nil`, where `activeAnimationName` is the `key` of the placeholder `CABasicAnimation` tracking the animation progress. My original thinking was that following this pattern with the `ExperimentalAnimationLayer` (and having both layers behave in the same way) would simplify things, but I think this is a pretty confusing / non-obvious way to do this so will probably change it.